README.md

```diff
@@ -3,3 +3,51 @@
 This repo host an initial code for the tasks performed during the Code Reviewers training.
 
 Application is based on this [Figma design](https://www.figma.com/file/t3EJUCAEViw3QasuJLPLVT/Microverse-Student-Potfolio-Templates-Main?node-id=1%3A1471).
+
+This is code review practice project.
+
+## Built With
+
```

Comment:
[Optional] 
- it would be nice if you have a live version where the project can be viewed by everybody